src/xdocs/checks/design/innertypelast.xml.template

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<document xmlns="http://maven.apache.org/XDOC/2.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/XDOC/2.0 https://maven.apache.org/xsd/xdoc-2.0.xsd">
+  <head>
+    <title>InnerTypeLast</title>
+  </head>
+  <body>
+    <section name="InnerTypeLast">
+      <p>Since Checkstyle 5.2</p>
+      <subsection name="Description" id="Description">
+        <p>
+          Checks nested (internal) classes/interfaces are declared at the bottom of the
+          primary (top-level) class after all init and static init blocks,
+          method, constructor and field declarations.
+        </p>
+      </subsection>
+
+      <subsection name="Examples" id="Examples">
+        <p id="Example1-config">
+          To configure the check:
+        </p>
+        <macro name="example">
+          <param name="path"
+                 value="resources/com/puppycrawl/tools/checkstyle/checks/design/innertypelast/Example1.txt"/>
+          <param name="type" value="config"/>
+        </macro>
+        <p id="Example1-code">Example:</p>
+        <macro name="example">
+          <param name="path"
+                 value="resources/com/puppycrawl/tools/checkstyle/checks/design/innertypelast/Example1.txt"/>
+          <param name="type" value="code"/>
+        </macro>
+      </subsection>
+
+      <subsection name="Example of Usage" id="Example_of_Usage">
+        <ul>
+          <li>
+            <a href="https://github.com/search?q=path%3Aconfig%20path%3A**%2Fcheckstyle-checks.xml+repo%3Acheckstyle%2Fcheckstyle+InnerTypeLast">
+            Checkstyle Style</a>
+          </li>
+        </ul>
+      </subsection>
+
+      <subsection name="Violation Messages" id="Violation_Messages">
+        <ul>
+          <li>
+            <a href="https://github.com/search?q=path%3Asrc%2Fmain%2Fresources%2Fcom%2Fpuppycrawl%2Ftools%2Fcheckstyle%2Fchecks%2Fdesign%20path%3A**%2Fmessages*.properties+repo%3Acheckstyle%2Fcheckstyle+%22arrangement.members.before.inner%22">
+            arrangement.members.before.inner</a>
+          </li>
+        </ul>
+        <p>
+          All messages can be customized if the default message doesn't suit you.
+          Please <a href="../../config.html#Custom_messages">see the documentation</a>
+          to learn how to.
+        </p>
+      </subsection>
+
+      <subsection name="Package" id="Package">
+        <p>
+          com.puppycrawl.tools.checkstyle.checks.design
+        </p>
+      </subsection>
+
+      <subsection name="Parent Module" id="Parent_Module">
```

Comment:
General comment: I wonder if this section is even needed now that we include the parent module in the config itself? I guess it is good to have the link to the parent module, just wanted to bring this up, though.

Reply:
> now that we include the parent module in the config itself

What do you mean by that?

Reply:
Before:
```
    <module name="InnerTypeLast"/>

```

After:
```
<module name="Checker">
  <module name="TreeWalker">
    <module name="InnerTypeLast"/>
  </module>
</module>

```

Reply:
Ohh yeah. I suppose it is the link that is important in the section here. Without it, the section is redundant.

Reply:
lets keep this section for now, I am not sure if this is good to remove it yet.
May be far later. User might be using it, I do not know.

Reply:
Yeah, I think we should keep it too, I thought it might be good to mention it and get other's thoughts.